### PR TITLE
48 update search page

### PR DESCRIPTION
**Pull request by chooseanother**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,12 @@ While doing development you might need to clera the database
 2. Run this command: `docker-compose -f local.yml run --rm django python manage.py shell`
 
 
+
+## If new static files are added
+this might be wrong
```
Comment (304125) [inline README.md:71]: maybe leave this line out??

File: msdb/msdb/static/css/project.css [hunk 2]
```diff
@@ -65,3 +65,18 @@
...
 }
+
+
+footer {
+  margin-top: 4em;
+}
+
+body{ 
+  min-height: 100vh;
+  display:flex;
```
Comment (304125) [inline msdb/msdb/static/css/project.css:76]: keep the styling consistent, add a space after colon
display: flex;
similar comment line 77 and 81

File: msdb/msdb/templates/base.html [hunk 3]
```diff
@@ -134,7 +134,7 @@
     </script>
     {% endcomment %}
     {% endblock inline_javascript %}
-
-    {% include "footer.html" %}
   </body>
+
+
```
Comment (304125) [inline msdb/msdb/templates/base.html:139]: generally very strange distribution of empty lines, but I would definitely not leave 2 empty lines

File: msdb/msdb/templates/movies/movie.html [hunk 4]
```diff
@@ -16,7 +16,7 @@ <h1>{{ movie.title }}</h1>
 <div class="container bg-dark text-white p-4">
   <div class="row">
     <div class="col-sm-3">
-      <img src="{{movie.poster_link}}" alt="{{movie.title}} poster" width="100%"/>
+      <img src="{{movie.poster_link}}" poster" width="100%"/>
```
Comment (304125) [inline msdb/msdb/templates/movies/movie.html:19]: i believe this is a mistake? why is alt removed and left with poster"?

File: msdb/msdb/templates/search/search.html [hunk 5]
```diff
@@ -24,11 +29,11 @@ <h5>{{ movie.title }}</h5>
...
     <div class="col-sm-6 col-md-3">
       <h2>People</h2>
-      {% for i in '0123456789kkkkk'|make_list %}
+      {% for person in people %}
         <a href="#" class="search_link">
           <div class="mb-4">
-            <h5 style="margin-bottom: 0;">Emma Stone</h5>
-            <p>Actor/Actress</p>
+            <h5 style="margin-bottom: 0;">{{ person.name }}</h5>
+            <p> {% if person.director %}Director {% endif %} {% if person.actor %}Actor/Actress{% endif %}</p>
```
Comment (304125) [inline msdb/msdb/templates/search/search.html:36]: add comma between Director and Actor/Actress

File: msdb/msdb/views/search.py [hunk 6]
```diff
@@ -10,11 +11,86 @@
...
+
+def search_movie(queries : list):
+    movies_result = Movie.objects.filter(reduce(operator.and_, (Q(title__icontains=x) for x in queries)))[:10]
+    # if no movies are found, do the search with or operator instead
+    if movies_result.count() == 0:
+        movies_result = Movie.objects.filter(reduce(operator.or_, (Q(title__icontains=x) for x in queries)))[:10]
+
+    return movies_result
+
+
```
Comment (304125) [inline msdb/msdb/views/search.py:39]: again, as above, I would generally stay away from keeping 2 empty lines (point for discussion though)

**Review by 304125 — CHANGES_REQUESTED** [review]

File: msdb/msdb/templates/search/search.html (hunk 5 above)
Comment (chooseanother) [reply msdb/msdb/templates/search/search.html:36]: Fixed

**Comment by chooseanother** [comment]
I'm used to not think about all those spaces, line length and empty lines because it is all handled by pre-commit /:

**Review by 304125 — APPROVED** [review]

**Pull request merged**